src/hotspot/cpu/x86/x86.ad

```diff
@@ -5797,6 +5806,49 @@ instruct signumD_reg(regD dst, regD zero, regD one, rRegP scratch, rFlagsReg cr)
   ins_pipe( pipe_slow );
 %}
 
+#ifdef _LP64
+instruct copySignF_reg(regF dst, regF src, regF tmp1, rRegL tmp2) %{
+  predicate(UseAVX >= 3 && VM_Version::supports_avx512vl());
+  match(Set dst (CopySignF dst src));
+  effect(TEMP tmp1, TEMP tmp2);
+  format %{ "CopySignF $dst, $src" %}
+  ins_encode %{
+    __ movl($tmp2$$Register, 0x7FFFFFFF);
+    __ vmovd($tmp1$$XMMRegister, $tmp2$$Register);
+    __ vpternlogd($dst$$XMMRegister, 0xE4, $src$$XMMRegister, $tmp1$$XMMRegister, Assembler::AVX_128bit);
+  %}
+  ins_pipe( pipe_slow );
+%}
+
+instruct copySignD_reg(regD dst, regD src, regD tmp1, rRegL tmp2, regD zero) %{
```

Comment:
Graph shape created for this will always pass zero operand as an immediate argument. Do you see any issue if you just keep one pattern (copySignD_imm) ?

Reply:
@mgkwill , haven't seen your response on this.

Reply:
@jatin-bhateja, @sviswa7 told me that you and her discussed it and decided that removing copySignD_reg was appropriate as it is not needed. I've removed copySignD_reg.

Reply:
Thanks! patch looks good otherwise.